docs/csharp/whats-new/csharp-13.md

```diff
@@ -76,6 +76,11 @@ You can read the details of the changes in the [proposal specification](~/_cshar
 The implicit "from the end" index operator, `^`, is now allowed in an object initializer expression. For example, you can now initialize an array in an object initializer as shown in the following code:
 
 ```csharp
+public class TimerRemaining
+{
+    public int[] buffer { get; set; } = new int[10];
```

Comment:
nit: the setter could be left out which would emphase the `buffer =` below isn't doing a set.